.github/workflows/testing.yml

```diff
@@ -125,6 +126,12 @@ jobs:
         # supported for use outside of 'steps'
         ports:
           - 9042:9042
+      dynamodb:
+        image: amazon/dynamodb-local
```

Comment:
This is container is unused

Reply:
should i delete or should i change it to something?

Reply:
just remove this block. Were testing it on aws instead of the container rn